frontend/src/components/course-learners-list/CourseLearnersList.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styles from './_course-learners-list.scss';
+import classNames from 'classnames/bind';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import { faCheck } from '@fortawesome/fontawesome-free-solid';
+
+let cx = classNames.bind(styles);
+
+class CourseLearnersList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayedData: [],
+      allDataDisplayed: false,
+    };
+    this.retrieveData = this.retrieveData.bind(this);
+  }
+
+  retrieveData = (amountToFetch) => {
+    const testData = [
+      {
+        firstName: 'Hans',
+        lastName: 'Gruber',
+        country: 'Germany',
+        dateEnrolled: '10-23-2017',
+        courseProgress: '59%',
+        courseCompleted: false,
+        dateCompleted: '',
+      },
+      {
+        firstName: 'John',
+        lastName: 'McClane',
+        country: 'USA',
+        dateEnrolled: '08-13-2016',
+        courseProgress: '100%',
+        courseCompleted: true,
+        dateCompleted: '12-04-2017',
+      },
+      {
+        firstName: 'Clane',
+        lastName: 'McJohn',
+        country: 'USA',
+        dateEnrolled: '08-13-2016',
+        courseProgress: '100%',
+        courseCompleted: true,
+        dateCompleted: '12-04-2017',
+      },
+    ]
+    console.log('Retrieving ', amountToFetch, ' records from the API.');
+    let temp = this.state.displayedData;
+    this.setState({
+      displayedData: temp.concat(testData.slice(temp.length, (temp.length + amountToFetch))),
+      allDataDisplayed: (amountToFetch >= testData.slice(temp.length).length),
+    })
+    console.log(this.state.displayedData);
+  }
+
+  componentDidMount() {
+    this.retrieveData(this.props.paginationMaxRows);
+  }
+
+
+  render() {
+    const learnersRender = this.state.displayedData.length ? this.state.displayedData.map((learner, index) => {
+      return (
+        <li key={index} className={styles['learner-row']}>
+          <span className={styles['first-name']}>{learner.firstName}</span>
```

Comment:
edx-platform only has fullname. it doesn't differentiate  first and last:  https://github.com/edx/edx-platform/blob/open-release/ginkgo.master/common/djangoapps/student/models.py#L242